src/Arcus.WebApi.Unit/Arcus.WebApi.Unit.csproj

```diff
@@ -12,23 +12,26 @@
         <RepositoryUrl>https://github.com/arcus-azure/arcus.webapi</RepositoryUrl>
         <PackageIconUrl>https://raw.githubusercontent.com/arcus-azure/arcus/master/media/arcus.png</PackageIconUrl>
         <GenerateDocumentationFile>true</GenerateDocumentationFile>
+        <OpenApiGenerateDocuments>false</OpenApiGenerateDocuments>
```

Comment:
Is there a reason why this is `false` while `GenerateDocumentationFile` is `true`?

Reply:
Yeah, this new one is added and only used in the .NET Core 3.0 version of swagger; but it gets an error when enabled because it can't find a build target with that name. Disabling it here will fix that. Maybe I should disable the one above that also?

Reply:
Ah it's the test project, it's ok.